{{cookiecutter.repo_name}}/setup.py

```diff
@@ -47,7 +47,9 @@ def get_version(*file_paths):
         '{{ cookiecutter.app_name }}',
     ],
     include_package_data=True,
-    install_requires=[{% if cookiecutter.models != "Comma-separated list of models" %}"django-model-utils>=2.0"{% endif %}],
+    install_requires=[
+        "Django>=1.8.15,<1.11"{% if cookiecutter.models != "Comma-separated list of models" %}, "django-model-utils>=2.0"{% endif %}
```

Comment:
The bottom of that range can probably just be 1.8 as in tox.ini, but I doubt if it makes that much of a difference.  Just a little concerned it could block installation on services which haven't upgraded to the latest Django 1.8.x recently.